recipes/py_gd/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "0.1.6" %}
+{% set sha256 = "7ac29a8664b3708fc7909687fbb5d73125b0807139f2c76d4f7f2ded0f121a87" %}
+{% set libgd_version = "2.2.2" %}
+
+package:
+    name: py_gd
+    version: {{ version }}
+
+source:
+    fn: py_gd-{{ version }}.tar.gz
+    url: https://github.com/NOAA-ORR-ERD/py_gd/archive/v{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+    # git_url: https://github.com/NOAA-ORR-ERD/py_gd.git
+    # git_tag: v{{ version }}
+
+build:
+    skip: True  # [py3k or win]
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
This is different kludge. We use it to control how compiled code gets built. In the future it should just be `toolchain`. Though we need to unpin some things first (which is a whole other story).